app/src/components/ModalBoard/ModalBoard.tsx

```diff
@@ -0,0 +1,170 @@
+import { useState, useContext, useRef, FormEvent, ChangeEvent } from "react";
+import BoardsContentContext from "@context/BoardsContent/BoardsContent";
+import BoardModalContext from "@context/BoardModal/BoardModal";
+import Modal from "@components/ui/Modal/Modal";
+import Input from "@components/ui/Input/Input";
+import Button from "@components/ui/Button/Button/Button";
+import styles from "./ModalBoard.module.scss";
+import ModalBoardsList from "./ModalBoardsList/ModalBoardsList";
+import { arrayMove } from "@dnd-kit/sortable";
+import { v4 as uuidv4 } from "uuid";
+import { DragEndEvent } from "@dnd-kit/core/dist/types/events";
+// POPRAW SCIEZKE na alias @shared
+import { Column } from "@shared/types";
+
+const ModalBoard = () => {
+  const [boardColumns, setBoardColumns] = useState<Column[]>([
+    { id: uuidv4(), value: "" },
```

Comment:
it won't work in this way with backend - we don't have API for as of now but as soon we have it will always come as part of response and should not be set on frontend side - so when you add new board you get in response id of item. But this is something we should not use for this case. I suggest to create simple counter which always increments on each input creation - even if input is removed then this id is not considered to reuse for new column input but next one (so counter always goes up and next input always have id + 1)